.github/workflows/build.yaml

```diff
@@ -27,4 +27,9 @@ jobs:
       - name: Create frontend package
         run: mvn -X -B -pl nflow-explorer install
       - name: Run the Maven test phase
-        run: mvn -X -B -P ${{ matrix.db }} test
+        run: mvn -X -B -P ${{ matrix.db }} -Dskip.npm -Dskip.bower -Dskip.grunt test
```

Comment:
We really want to skip all of these?

Reply:
Yes. We already `install` the frontend package in the previous step. These only run `test`. These options thus this skips the double build of frontend.